code/modules/mob/living/carbon/human/human.dm

```diff
@@ -85,6 +85,8 @@
 			if(mind.changeling)
 				stat("Chemical Storage", "[mind.changeling.chem_charges]/[mind.changeling.chem_storage]")
 				stat("Absorbed DNA", mind.changeling.absorbedcount)
+			if(mind.umbrage_psionics)

```

Comment:
I have no idea how to work with UI elements, but it's planned by the end of the project.